### DISCUSSION TITLE
🚀 Include an "any" Status parameter option

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#4220] [Feature Requests]
### Suggest an idea for ExpressionEngine

I'd like to display some Entries, regardless of which Status they might be set to.
In my ecommerce sites (typically) I have many Status options (Paid, Shipped, Refunded, etc) but often I'd like to display orders regardless of their Status. In a Channel Form driven Dashboard for example.

Having this parameter available in the exp:channel:entries loop would be really useful:
status="any"

**Comment by intoeetive** [comment]
@ic360-com the common workaround for this is `status="not nonexistingstatus"`, but I agree it can be made more straightforward
